articles/101_build_tool.md

```diff
@@ -183,32 +195,50 @@ After cloning the repositories containing Gazebo and all its dependencies (exclu
 Meta information not inferable from the sources can be provided externally without adding or modifying any files in the workspace.
 After the build a single file can be sourced / invoked to setup the environment to use Gazebo (e.g. `GAZEBO_MODEL_PATH`).
 
+### Individual features
+
+The following describes features mostly already present in the present build tools, to explain why these tools are complex to merge.
+
 #### Mixing different build systems
 
-The build tool will support using different build systems within a single workspace.
-If these packages inter-operate with each other correctly depends also to a large degree on the build system.
-The build tool should ensure that it doesn't prevent that use case.
+The build tool will support using different build systems within a single workspace, as ament_tools or catkin_tools already do.
+But there may be restrictions on which build systems can be combined in which kind of workspace (e.g. building catkin packages may not be possible in an ament workspace).
+Removing such restrictions will be an ongoing process.
 
-### Software Criteria
+#### Build Environment Setup
```

Comment:
This is not an individual feature but a strict requirement in order to build package automatically on-top of each other. Therefore I don't think it should be moved that far down.